File: Learning Requirements/DockerServer From Scratch/src/main/java/Network/ClientSide/uploadedCode/uploadedCode.cpp

#include <iostream>
#include <time.h>
#include <cstdlib>
#include <string>

int main () { 

	srand( time( NULL ) );
	std::string response;

	while ( true ) {

		std::cin >> response; 
		if ( response == "END" ) 
			exit( 0 );
		else if ( response == "ENTER_A_NUMBER" ) 
			std::cout << ( ( rand() % 6 ) + 1 ) << std::endl;
		else
			exit( 1 );

	}

	return 0;

}

File: Learning Requirements/DockerServer From Scratch/src/main/java/Network/ClientSide/uploadedCode/uploadedCode.py

import random
import time
import sys

random.seed(time.time)

respone = ""

while (True):
	
	response = input()
	if response == "END":
		sys.exit(0)
	elif response == "ENTER_A_NUMBER":
		print(random.randint(1, 6))
	else:
		sys.exit(1)

